django_app/settings.py

```diff
@@ -188,6 +190,9 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.1/howto/static-files/
 
+
+STATICFILES_STORAGE = "whitenoise.storage.CompressedManifestStaticFilesStorage"
```

Comment:
there is missing STATIC_ROOT variable because on production server can not look into every app static directory so need to collect all the static in single directory

Reply:
do you mean this?
![image](https://user-images.githubusercontent.com/28830125/219039877-0b378290-ae37-4765-aa82-9c4028ed5230.png)